src/app/analytics/analytics.factory.js

```diff
@@ -150,7 +191,12 @@ const analyticsFactory = /* @ngInject */ function ($window, $timeout, sessionSer
       try {
         // Build products variable
         this.buildProductVar(cartData)
-
+        // Send GTM Advance Ecommerce event
+        if (typeof $window.dataLayer !== 'undefined') {
```

Comment:
So, `cartView` isn't currently called anywhere. Maybe if you aren't going to use the `cartData` parameter, it should be removed from the definition?